mapr/datadog_checks/mapr/utils.py

```diff
@@ -1,11 +1,10 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-
 import socket
 
 
-def get_stream_id_for_topic(topic_name, rng=2):
+def get_stream_id_for_topic(topic_name, rng=1):
```

Comment:
Note for reviewer: Even though this function supports multiple `rng` values, we MapR default value since 6.1 si to have a single stream for all topics.

Reply:
^ that should be a in code comment